src/screens/Dashboard/screens/Login/constants/index.ts

```diff
@@ -0,0 +1,7 @@
+export const dataForm = [
```

Comment:
- try to be a little more specific with the names of the constants to export, also remember that the constants are exported in upper case
```suggestion
export const SIGN_UP_FIELDS = [
```